lowcar/devices/KoalaBear/KoalaBear.cpp

```diff
@@ -28,24 +28,33 @@
 uint16_t ctrl_read_data;
 uint16_t torque_read_data;
 
-volatile int32_t enc_a, enc_b;
+//******************************* KOALABEAR CONSTANTS AND PARAMS ****************************//
+
+// ----------------------> CHANGE THIS TO IMPLEMENT MAX SPEED FOR KARTS <----------------------
+#define MAX_DUTY_CYCLE 0.5 // maximum duty cycle to cap how fast the motors can go: range (0, 1] 
+// --------------------------------------------------------------------------------------------
+
+// default values for PID controllers
+#define KP_DEFAULT 0.035
+#define KI_DEFAULT 0.005
+#define KD_DEFAULT 0.0
```

Comment:
Would it be good to add a small comment to each of these variables or description above on what each of these PID values represent? Or maybe a mention on how the wiki covers what they are(even though its not super in depth there and might need to be expanded on)? 
Also out of curiosity why are these the default values? I know KD has to be zero since there has obviously been no movement whatsoever so you cant calculate the change in error, but what about the other two?

Reply:
Yes, I think I'm going to add a comment saying that the PID controller is talked about in the wiki... as far KD being zero actually that is a little unusual, but for our motors it seems that it's a little too responsive to it so I set to 0 for the time being!